packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -800,6 +804,645 @@ let rows = [
+}
+```
+
+</details>
+
+### Resizable table body
+
+Similar to `TableRowGroup` and `TableHeaderRow`, `TableRow` and `TableCell` only require minor style changes to
+accommodate the new `display` changes made in the `ResizableColumnsTable`. The changes to `TableRow` are made to extend the
+background of the row to the full width of its child cells. `TableCell` now receives its width from <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useTableColumnResizeState} />
+and handles text overflow.
+
+```tsx example export=true render=false
+function ResizableTableRow({item, children, state}) {
+  // Same as previous TableRow implementation
+  ///- begin collapse -///
+  let ref = useRef();
+  let isSelected = state.selectionManager.isSelected(item.key);
+  let {rowProps, isPressed} = useTableRow({
+    node: item
+  }, state, ref);
+  let {isFocusVisible, focusProps} = useFocusRing();
+  ///- end collapse -///
+  return (
+    <tr
+      style={{
+        background: isSelected
+          ? 'blueviolet'
+          : isPressed
+            ? 'var(--spectrum-global-color-gray-400)'
+            : item.index % 2
+              ? 'var(--spectrum-alias-highlight-hover)'
+              : 'none',
+        color: isSelected ? 'white' : null,
+        outline: 'none',
+        boxShadow: isFocusVisible ? 'inset 0 0 0 2px orange' : 'none',
+        /*- begin highlight -*/
+        display: 'flex',
+        width: 'fit-content'
+        /*- end highlight -*/
+      }}
+      {...mergeProps(rowProps, focusProps)}
+      ref={ref}>
+      {children}
+    </tr>
+  );
+}
+```
+
+```tsx example export=true render=false
+function ResizableTableCell({cell, state, widths}) {
+  // Same as previous TableCell implementation
+  ///- begin collapse -///
+  let ref = useRef();
+  let {gridCellProps} = useTableCell({node: cell}, state, ref);
+  let {isFocusVisible, focusProps} = useFocusRing();
+  ///- end collapse -///
+  /*- begin highlight -*/
+  let column = cell.column;
+  /*- end highlight -*/
+
+  return (
+    <td
+      {...mergeProps(gridCellProps, focusProps)}
+      style={{
+        padding: '5px 10px',
+        cursor: 'default',
+        outline: 'none',
+        boxShadow: isFocusVisible ? 'inset 0 0 0 2px orange' : 'none',
+        /*- begin highlight -*/
+        width: widths.get(column.key),
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+        display: 'block',
+        flex: '0 0 auto',
+        boxSizing: 'border-box'
+        /*- end highlight -*/
+      }}
+      ref={ref}>
+      {cell.rendered}
+    </td>
+  );
+}
+```
+
+And with that, all necessary changes to the previous table implementation have been made and we now have a table that supports resizable columns!
+The example below supports resizing via mouse, keyboard, touch, and screen reader interactions. To see an example with sorting and selection, see the styled example!
```

Comment:
Styled example in progress here: https://codesandbox.io/s/agitated-darwin-uki38i?file=/src/Table.tsx. I think the nightly is causing issues due to dupe packages, still need to investigate fully